### PR TITLE
JAMES-2406 Play Cassandra SMTP MPT tests on a SMTP only server

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraObjectStoreModule.java [hunk 1]
```diff
@@ -35,5 +37,10 @@ protected void configure() {
 
         bind(ObjectStore.class).to(CassandraBlobsDAO.class);
         bind(BlobId.Factory.class).to(CassandraBlobId.Factory.class);
+
+
+
```
Comment (rouazana) [inline server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraObjectStoreModule.java]: too much cr/lf

Comment (chibenwa) [reply server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraObjectStoreModule.java]: Ahh

File: server/container/guice/cassandra-guice/src/test/java/org/apache/james/modules/CassandraTestModule.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import com.datastax.driver.core.Session;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.name.Names;
+
+public class CassandraTestModule extends AbstractModule {
+    private final String cassandraHost;
+    private final int cassandraPort;
+
+    public CassandraTestModule(String cassandraHost, int cassandraPort) {
```
Comment (aduprat) [inline server/container/guice/cassandra-guice/src/test/java/org/apache/james/modules/CassandraTestModule.java]: You can use a _org.apache.james.util.Host_ parameter.

Comment (chibenwa) [reply server/container/guice/cassandra-guice/src/test/java/org/apache/james/modules/CassandraTestModule.java]: Pff...

Comment (chibenwa) [reply server/container/guice/cassandra-guice/src/test/java/org/apache/james/modules/CassandraTestModule.java]: Let's refactor!

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
```
[c766c7e03c56f2b0438eae0d281f262973546ace] [ERROR] /james-project/mpt/impl/smtp/cassandra/src/test/java/org/apache/james/mpt/smtp/CassandraSmtpStarttlsCommandTest.java:[40,17] constructor SmtpTestModule in class org.apache.james.mpt.smtp.SmtpTestModule cannot be applied to given types;
[c766c7e03c56f2b0438eae0d281f262973546ace]   required: org.apache.james.mpt.smtp.SmtpTestModule.Port,org.apache.james.util.Host
[c766c7e03c56f2b0438eae0d281f262973546ace]   found: org.apache.james.mpt.smtp.SmtpTestModule.Port,java.lang.String,int
[c766c7e03c56f2b0438eae0d281f262973546ace]   reason: actual and formal argument lists differ in length
[c766c7e03c56f2b0438eae0d281f262973546ace] [ERROR] /james-project/mpt/impl/smtp/cassandra/src/test/java/org/apache/james/mpt/smtp/CassandraForwardSmtpTest.java:[40,17] constructor SmtpTestModule in class org.apache.james.mpt.smtp.SmtpTestModule cannot be applied to given types;
[c766c7e03c56f2b0438eae0d281f262973546ace]   required: org.apache.james.mpt.smtp.SmtpTestModule.Port,org.apache.james.util.Host
[c766c7e03c56f2b0438eae0d281f262973546ace]   found: org.apache.james.mpt.smtp.SmtpTestModule.Port,java.lang.String,int
[c766c7e03c56f2b0438eae0d281f262973546ace]   reason: actual and formal argument lists differ in length
```

**Comment by mbaechler** [comment]
test this please

**Comment by rouazana** [comment]
```
[774d68ac61f3883da311b46ff8310b90528770dd] [ERROR] COMPILATION ERROR : 
[774d68ac61f3883da311b46ff8310b90528770dd] [INFO] -------------------------------------------------------------
[774d68ac61f3883da311b46ff8310b90528770dd] [ERROR] /james-project/mpt/core/src/main/java/org/apache/james/mpt/user/ScriptedUserAdder.java:[106,81] incompatible types: int cannot be converted to org.apache.james.util.Port
```

**Comment by rouazana** [comment]
OMG, ant tests.
```
[31aa3f2f449a0400ef91d1c97a79c1321dad8c8f] main:
[31aa3f2f449a0400ef91d1c97a79c1321dad8c8f] 
[31aa3f2f449a0400ef91d1c97a79c1321dad8c8f] testMustSetPort:
```

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please (:green_apple:  but some JUNIT 5 tests were skept)

**Comment by chibenwa** [comment]
```
[a7c481b1bd3896194d95994ea7b60f4942e926c5] Tests in error: 
[a7c481b1bd3896194d95994ea7b60f4942e926c5]   ToRepositoryTest.userShouldBeAbleToAccessReprocessedMails:134 » ConditionTimeout
[a7c481b1bd3896194d95994ea7b60f4942e926c5] 
[a7c481b1bd3896194d95994ea7b60f4942e926c5] Tests run: 138, Failures: 0, Errors: 1, Skipped: 0
```

test this please

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**